### PR TITLE
Review and integrate UI enhancements

### PR DESCRIPTION
**Pull request by charlesh88**
See #2000 for details on changes and testing notes.

### Author Checklist

1. Changes address original issue? Y
1. Unit tests included and/or updated with changes? N/A
1. Command line build passes? Y
1. Changes have been smoke-tested? Y

### CODE REVIEW

File: platform/commonUI/general/res/sass/_inspector.scss [hunk 1]
```diff
@@ -50,83 +42,194 @@
...
-        padding: $interiorMarginSm 0;
-        .label {
-            color: $colorInspectorPropName;
-            text-transform: uppercase;
-        }
-        .value {
-            color: $colorInspectorPropVal;
-            word-break: break-all;
-        }
+    //.pane-header {
```
Comment (akhenry) [inline platform/commonUI/general/res/sass/_inspector.scss]: We should delete all the commented styles in this file

File: platform/features/conductor/core/res/templates/time-conductor.html [hunk 2]
```diff
@@ -77,7 +77,7 @@
                     </span>
                 </span>
 
-                <input type="submit" class="off">
+                <input type="submit" class="off hide">
```
Comment (akhenry) [inline platform/features/conductor/core/res/templates/time-conductor.html]: `display: none` breaks form submission in some browsers. See https://github.com/nasa/openmct/pull/2020#issuecomment-387827362 . Is there another approach we can take here?

File: src/plugins/plot/res/templates/plot-options-edit.html [hunk 3]
```diff
@@ -19,95 +19,197 @@
...
             </li>
         </ul>
-    </ul>
-    <form class="inspector-config"
+    </div>
+
+
+
+
+<!--
```
Comment (akhenry) [inline src/plugins/plot/res/templates/plot-options-edit.html]: We should remove this commented code

File: platform/commonUI/general/res/sass/_inspector.scss [hunk 4]
```diff
@@ -50,83 +42,194 @@
...
-        border-radius: $basicCr;
-        background-color: $colorInspectorSectionHeaderBg;
-        color: $colorInspectorSectionHeaderFg;
+.l-inspect,
+.l-inspect table tr td {
+    font-size: 0.75rem;
+}
+
+.l-inspect {
+    //@extend .abs;
```
Comment (akhenry) [inline platform/commonUI/general/res/sass/_inspector.scss]: We should remove this commented code

File: platform/commonUI/general/res/sass/_inspector.scss (hunk 1 above)
Comment (charlesh88) [reply platform/commonUI/general/res/sass/_inspector.scss]: Missed that, thanks. I'll do a sweep and make sure I didn't miss any other commented styles.

File: platform/features/conductor/core/res/templates/time-conductor.html (hunk 2 above)
Comment (charlesh88) [reply platform/features/conductor/core/res/templates/time-conductor.html]: Ah, wondered why this was there, should have known. Yes, think I can set to 0 height and width or similar.

File: platform/commonUI/general/res/sass/_inspector.scss (hunk 1 above)
Comment (charlesh88) [reply platform/commonUI/general/res/sass/_inspector.scss]: Good god I left a lot of dirty laundry laying about. I feel ashamed, but have cleaned up the dorm room.

File: src/plugins/plot/res/templates/plot-options-edit.html (hunk 3 above)
Comment (charlesh88) [reply src/plugins/plot/res/templates/plot-options-edit.html]: Removed.

File: platform/commonUI/general/res/sass/_inspector.scss (hunk 4 above)
Comment (charlesh88) [reply platform/commonUI/general/res/sass/_inspector.scss]: Removed.

File: platform/features/conductor/core/res/templates/time-conductor.html (hunk 2 above)
Comment (charlesh88) [reply platform/features/conductor/core/res/templates/time-conductor.html]: Fixed.

**Comment by charlesh88** [comment]
@akhenry Thanks Andrew. Sorry about the dirty laundry strewn about. All comments should be addressed at this point, and conflicts resolved.

**Comment by charlesh88** [comment]
@larkin Conflicts resolved.

**Comment by larkin** [comment]
# Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A, style only
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y

**Pull request merged**